### PR TITLE
doc: sample app

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Fixes NAN-762

- Add a link in the sidebar and a small doc to introduce the app
Feel free to correct my bad english 👍🏻 

Screenshot 2024-04-18 at 01 14 12

### CODE REVIEW

**Review by TBonnin — COMMENTED** [review]
looks good. The sample app repo is private though. 
Also what happen to the sample app codename? 🐺

**Review by TBonnin — APPROVED** [review]

File: docs-v2/images/github.svg
Comment (TBonnin) [inline docs-v2/images/github.svg:1]: what is this logo doing in this PR?

**Comment by bodinsamuel** [comment]
> looks good. The sample app repo is private though.

Yes I will put it public once approved

> Also what happen to the sample app codename? 🐺

We agreed on choosing something clearer :p

File: docs-v2/images/github.svg
Comment (bodinsamuel) [reply docs-v2/images/github.svg:1]: I added it for the button

File: docs-v2/sample-app.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (bastienbeurier) [inline docs-v2/sample-app.mdx:1]: Proposal

```
---
title: 'Sample Application'
sidebarTitle: 'Sample App'
description: 'Learn how to integrate Nango using our example application'
---

Jumpstart your integration with Nango by exploring our sample application, which includes a front-end, a back-end, and integrations.

## Highlights

- **`front-end/` folder**: Features a frontend interface where users can connect to integrations and visualize external data.

- **`back-end/` folder**: Includes a backend API that receives Nango's [webhooks](/understand/concepts/webhooks), fetches the latest data from the Nango API, persists it in a database, and powers the frontend.

- **`nango-integrations/` folder**: Contains integration scripts that are deployed to Nango using the [CLI](/reference/cli), syncing data between Nango and external APIs.

## Access the repository

<a href="https://github.com/NangoHQ/sample-app" target="_blank" className="inline-flex items-center gap-2 border border-gray-400 h-7 px-4 py-5">
  <img src="/images/github.svg" alt="GitHub logo" className="w-5" />
  <div>NangoHQ/sample-app</div>
</a>

<Tip>
**Questions, problems, feedback?** Please reach out in the [Slack community](https://nango.dev/slack).
</Tip>
```

Comment (bodinsamuel) [reply docs-v2/sample-app.mdx:1]: Changed 👌🏻

**Review by bastienbeurier — APPROVED** [review]

**Pull request merged**